content/agent/configuration/configure-nginx-agent-group.md

```diff
@@ -13,7 +13,7 @@ doctypes: ["task"]
 
 During installation, NGINX Agent detects the NGINX user (typically `nginx`) for the master and worker processes and adds this user to a group called `nginx-agent`.
 
-If you change the NGINX username after installing the NGINX Agent, you'll need to add the new username to the `nginx-agent` group so that the NGINX socket has the proper permissions.
+If you change the NGINX username after installing NGINX Agent, you'll need to add the new username to the `nginx-agent` group so that NGINX socket has the proper permissions.
```

Comment:
```suggestion
If you change the NGINX username after installing NGINX Agent, you'll need to add the new username to the `nginx-agent` group so that the NGINX socket has the proper permissions.
```

`the NGINX socket` is correct. Only remove `the` from the product.